sdk/communication/Azure.Communication.Common/src/CommunicationIdentifier.cs

```diff
@@ -73,6 +79,11 @@ public static CommunicationIdentifier FromRawId(string rawId)
                 "8:dod:" => new MicrosoftTeamsUserIdentifier(suffix, false, CommunicationCloudEnvironment.Dod),
                 "8:gcch:" => new MicrosoftTeamsUserIdentifier(suffix, false, CommunicationCloudEnvironment.Gcch),
                 "8:acs:" or "8:spool:" or "8:dod-acs:" or "8:gcch-acs:" => new CommunicationUserIdentifier(rawId),
+                "28:gcch-global:" => new MicrosoftBotIdentifier(suffix, true, CommunicationCloudEnvironment.Gcch),
```

Comment:
Thanks! Good point!

Reply:
I was a bit reserved regarding this change. One one hand it reduces code duplication, on the other hand it reduces readability....it's about finding the balance between the two. But ok, since you've already went on with it, let's keep it this way.
And I think we were discussing it with @AikoBB the first time we introduced the From/ToRawID functionality and saying something along the lines that if we add more identifiers, we'll refactor it this way.